project_script/main.py

```diff
@@ -104,8 +102,49 @@ def integrate_data():
 
 density_of_carriers = [element * CHARGE_DENSITY_CALCULATION for element in integration_results]
 
-pd.DataFrame(density_of_carriers).to_csv(
-    "C:/Users/robee/Desktop/density_of_carriers_test.txt",
-    sep='\t',
-    decimal=','
-    )
+# pd.DataFrame(density_of_carriers).to_csv(
+#     "C:/Users/robee/Desktop/density_of_carriers_test.txt",
+#     sep='\t',
+#     decimal=',',
+#     )
+
+odd_columns_subtracted_smoothed_transposed = np.array(dt.smooth(odd_columns_subtracted_transposed_in_arrays))
+
+peaks = dt.find_the_peaks(odd_columns_subtracted_smoothed_transposed)
+
+indexes_list = dt.find_peak_index(odd_columns_subtracted_smoothed_transposed, peaks)
+
+time_max_values = dt.flatten(dt.find_time_max(time_photo_celiv_ramp_time_transposed, indexes_list))
+
+peak_values = pd.DataFrame([peaks, time_max_values]).transpose().set_axis(['current peak', 'peak time'], axis=1)
+
+peak_values['first term of mobility calculations'] = cv.DEVICE_THICKNESS ** 2 / (2 * peak_values['peak time'] ** 2) \
```

Comment:
I never seen a string index like this, it's common for you to have `peak_values[0]` or `peak_values['first_term`], `peak_values['mobility']`, but not a sentence as a index. Check if it's right, or if you could improve this, pls.

Reply:
Hi, sorry for the late!

it's all right 'cause it's not related to any index. In this case, it's something 'bout the Pandas library: the call
DataFrame['something'] = values is translated to pandas as a column with header something and content value in each line, that is, in the case above I am not meaning about an index, I am meaning about the title of the column that is formed by the expression.